word/tool.gpt

```diff
@@ -0,0 +1,43 @@
+---
+Name: Word
+Description: Tools for interacting with Microsoft Word documents
+Metadata: bundle: true
+Share Tools: List Docs, Get Doc
+
+---
+Name: List Docs
+Description: List all word documents available to the user
```

Comment:
nit - i believe capitalization matters to the LLM to some extent, so all usages of `Word` as in `MS Word` should have the same capitalization:

```suggestion
Description: List all Word documents available to the user
```

Reply:
I changed the capitalization and used `Microsoft Word` more ubiquitously.

Only question is whether the name of the category/bundle should change from `Word` to `Microsoft Word`, and in that case should I restructure the directories to align with the convention we're using for google products; e.g. `{word|excel|outlook}` -> `microsoft/{word|excel|outlook}`?

I know Darren was looking to keep the directory structure as flat as possible.

Reply:
i dont know if darren would agree, but i think that the directory structure should match googles, and it should be something like `microsoft365` to reflect the fact that these tools are for m365 only, not local documents. i wouldnt make any changes on that without passing it by darren or craig. at the very least though we should standardize, so either this or google should be changed to match at some point. i wouldnt let it hold up this pr.

Reply:
cool, I'll merge this then follow up in slack with our discussion